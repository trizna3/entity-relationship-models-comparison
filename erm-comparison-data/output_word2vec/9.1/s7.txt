fight model-EMPTY
rocket type-EMPTY
airplane-airplanes
transport model-EMPTY
marine role-EMPTY
marine-soldiers
airplane model-EMPTY
