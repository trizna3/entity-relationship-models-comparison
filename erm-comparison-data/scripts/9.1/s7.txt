entity set airplanes {
	"serial number"
	"name"
};

entity set "fighter planes" {
};

entity set "transport planes" {
	"person capacity"
	"material volume"
	"material weight"
};


entity set rockets {
	"type"
};

entity set soldiers {
	"name"
	"actual health"
	"maximal health"
	"speed"
	"strength"
	"agility"
	"inteligence"
};

entity set roles {
	"role name"
	"modifier"
};


association "can carry" between 
{
	"fighter planes" MANY
	rockets MANY
}{
	"count"
};

association between 
{
	airplanes MANY
	soldiers MANY
	roles MANY
}{};

generalization from "fighter planes" to "airplanes";

generalization from "transport planes" to "airplanes";
