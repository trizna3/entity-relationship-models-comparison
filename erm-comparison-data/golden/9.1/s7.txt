airplane model-EMPTY
fight model-fighter planes
rocket type-rockets
transport model-transport planes
airplane-airplanes
marine-soldiers
marine role-roles